consensus/src/block_storage/tracing.rs

```diff
@@ -20,7 +20,7 @@ impl BlockStage {
     pub const QC_AGGREGATED: &'static str = "qc_aggregated";
     pub const RAND_ADD_DECISION: &'static str = "rand_add_decision";
     pub const RAND_ADD_SHARE: &'static str = "rand_add_share";
-    pub const RAND_AGG_DECISION: &'static str = "rand_agg_decision";
+    // pub const RAND_AGG_DECISION: &'static str = "rand_agg_decision";
```

Comment:
Why did we remove this?

Reply:
unused. commenting it out was just a double check that everything compiles without it. find-all shows no other 'rand_agg_decision'

Reply:
later work on `main` made this a merge conflict because they fixed it there.